pkgs/applications/video/obs-studio/wlrobs.nix

```
@@ -0,0 +1,38 @@
+# (the following is somewhat lifted from ./linuxbrowser.nix)
+# We don't have a wrapper which can supply obs-studio plugins so you have to
+# somewhat manually install this:
+
+# nix-env -f . -iA obs-wlrobs
+# mkdir -p ~/.config/obs-studio/plugins/wlrobs/bin/64bit
+# ln -s ~/.nix-profile/share/obs/obs-plugins/wlrobs/bin/64bit/libwlrobs.so ~/.config/obs-studio/plugins/wlrobs/bin/64bit
+{ stdenv, fetchhg, wayland, obs-studio }:
+stdenv.mkDerivation {
+  pname = "obs-wlrobs";
+  version = "20191008";
+
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/wlrobs";
+    rev = "82e2b93c6f662dfd9d69f7826c0096bef585c3ae";
+    sha256 = "1d2mlybkwyr0jw6paamazla2a1cyj60bs10i0lk9jclxnp780fy6";
+  };
+
+  buildInputs = [ wayland obs-studio ];
+
+  preBuild = ''
+    cd Release
+  '';
```

alternative would be 
```suggestion
  sourceRoot  = "hg-archive/Release";
```
not sure which one is better :(

---

I think the source isn't in `hg-archive/Release`, and just the build tooling lives in that dir. That is why I picked `cd` instead... Not sure.

---

I changed it during the review, and it was able to build. But I don't feel super strong about this. :)